libc-test/build.rs

```diff
@@ -4473,7 +4474,11 @@ fn test_linux(target: &str) {
         // `anonymous_1` is an anonymous union
         (struct_ == "ptp_perout_request" && field == "anonymous_1") ||
         // `anonymous_2` is an anonymous union
-        (struct_ == "ptp_perout_request" && field == "anonymous_2")
+        (struct_ == "ptp_perout_request" && field == "anonymous_2") ||
+        // FIXME: `adjust_phase` requires >= 5.7 kernel headers
+        // FIXME: `max_phase_adj` requires >= 5.19 kernel headers
```

Comment:
I know we don't do this very many places but I'm trying to make sure all our fixmes are greppable to something specific

```suggestion
        // FIXME(linux): `adjust_phase` requires >= 5.7 kernel headers
        // FIXME(linux): `max_phase_adj` requires >= 5.19 kernel headers
```